python/python/Ice/CompressBatch.py

```diff
@@ -6,13 +6,15 @@ class CompressBatch(EnumBase):
     """
     The batch compression option when flushing queued batch requests.
 
```

Comment:
Fixed this to be consistent with the generated code.

`Enumerators` isn't a 'standard' heading, and as far as I can tell, there is no convention or standard for writing the type (`CompressBatch`) after each enumerator. This seems to be something we invented (or copied from another language...)